website/content/en/docs/deployment/cognito/guide-automated.md

```diff
@@ -118,10 +124,13 @@ This guide assumes you have Python 3.8 installed and that you have completed the
     kubectl delete profiles --all
     kubectl delete ingress -n istio-system istio-ingress
     ```
+
 1. Delete the kubeflow deployment
-   ```bash
-    kustomize build deployments/cognito | kubectl delete -f -
-    ```
+> Note: Make sure you have the correct INSTALLATION_OPTION and DEPLOYMENT_OPTION environment variables set for your chosen installation.
```

Comment:
either add an echo statement for verification OR create a tab pane similar to install